### PR TITLE
fix: Listing of runs to use standard DRF accessors

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Batch exports runs has overrides that don't hook into our standard model very well

## Changes

* Fixes this to ensure it only does a simple override for filtering

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

File: posthog/batch_exports/http.py [hunk 1]
```diff
@@ -95,35 +95,21 @@ class BatchExportRunViewSet(TeamAndOrgViewSetMixin, viewsets.ReadOnlyModelViewSe
     queryset = BatchExportRun.objects.all()
     serializer_class = BatchExportRunSerializer
     pagination_class = RunsCursorPagination
+    filter_rewrite_rules = {"team_id": "batch_export__team_id"}
```
Comment (benjackwhite) [inline posthog/batch_exports/http.py:98]: @tomasfarias this is how you can ensure the team filtering works for models that don't themselves have a `team_id`

File: posthog/batch_exports/http.py [hunk 2]
```diff
@@ -95,35 +95,21 @@ class BatchExportRunViewSet(TeamAndOrgViewSetMixin, viewsets.ReadOnlyModelViewSe
...
-            return self.queryset.filter(batch_export_id=self.kwargs["parent_lookup_batch_export_id"]).order_by(
-                "-created_at"
-            )
-
-    def list(self, request: request.Request, *args, **kwargs) -> response.Response:
-        """Get all BatchExportRuns for a BatchExport."""
-        if not isinstance(request.user, User) or request.user.team is None:
-            raise NotAuthenticated()
+    def get_queryset(self):
+        queryset = super().get_queryset()
```
Comment (benjackwhite) [inline posthog/batch_exports/http.py:101]: We always want to call the super value unless we are really going off piste (for example doing CH queries or something

File: posthog/batch_exports/http.py [hunk 3]
```diff
@@ -342,9 +328,6 @@ class BatchExportViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     serializer_class = BatchExportSerializer
 
     def get_queryset(self):
-        if not isinstance(self.request.user, User):
```
Comment (benjackwhite) [inline posthog/batch_exports/http.py:345]: We don't need to check authentication - thats what the TeamAndOrgViewSetMixin is responsible for.

**Review by tomasfarias — APPROVED** [review]

**Review by Twixes — COMMENTED** [review]
Post-humous ✅ :shipit:

**Pull request merged**